### ISSUE TITLE
Catch-22 with class containing nothing but protected or private methods

### ISSUE THREAD

**Issue by lee-dohm** [rubocop/rubocop#600]
I have a subclass in my current Rails project that looks approximately like so:

``` ruby
# Test class.
class Foo
  private

  def bar
    puts 'bar'
  end
end
```

For which I now (on `master`: 7b8e9ab60d669d4c26def099c7fe9dd7a2724582 ) get the warning:

```
C: Keep a blank line before and after private.
```

But if I do as it says and add the blank line before `private`, I get the following message:

```
C: Extra blank line detected at body beginning.
```

I think the `EmptyLinesAroundAccessModifier` cop should be relaxed by not requiring a blank line before the access modifier if it is the first semantic element in the class body.
